interview-questions.md

```diff
@@ -4,49 +4,60 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. Why would you use the method super()?
 
-  Your answer:
+  Your answer: super() is used to call upon the constructor() of the parent class. So inside a child class, it will have a super() to refer to all the properties of the parent class and you are able to generate more properties inside your child class.
 
-  Researched answer:
+  Researched answer: You would use the method super() to access and call functions from an object's parent. The method super() will make you child class inherit the methods from the parent class's constructor() method, therefor, the child has access to the parent's properties and methods.
 
 
 
 2. What is a virtual DOM?
 
-  Your answer:
+  Your answer: I think that a virtual DOM is possibly a representation of the actual DOM. So instead of actually manipulating a DOM directly, we manipulate the virtual DOM
```

Comment:
You got it! I think it would be wise to give an example of how a virtual DOM is used and why its useful.